internal/mesherytest/README.md

```
@@ -0,0 +1,226 @@
+only and dies under emulation on arm64 during content seeding. The source builds
+and runs natively, and Meshery's `go.mod` targets Go 1.26.4:
+
+```bash
+cd meshery/server/cmd && go build -o meshery-server .
+PORT=9081 PROVIDER=Local KEYS_PATH=../../server/permissions/keys.csv ./meshery-server
+```
+
+It seeds itself with 355 designs and 292 models, which is enough to exercise
+pagination and the design endpoints for real. Confirmed:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)